src/service/viewer-impl.js

```diff
@@ -790,15 +791,26 @@ export class Viewer {
   }
 
   /**
-   * Retrieves the Base CID from the viewer
+   * Get/set the Base CID from/to the viewer.
+   * @param {string=} opt_data Stringified JSON object {cid, time}.
    * @return {!Promise<string|undefined>}
    */
-  getBaseCid() {
+  baseCid(opt_data) {
```

Comment:
Exactly. `Viewer.baseCid` is compiled in core `v0.js`. The calling client is in `cid-impl.js` that's compiled into `analytics-0.1.js`. So, at least temporary skew is possible between these two files. The probability is, of course, very low since we handle skew on our CDN, so if you feel like this is overkill, feel free to ignore.